src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/Utilities.cs

```diff
@@ -40,5 +42,15 @@ public static KeyValueLogScope CreateArtifactScope(string artifactType, IEndpoin
         {
             return (!pid.HasValue && !uid.HasValue && string.IsNullOrEmpty(name)) ? null : new ProcessKey(pid, uid, name);
         }
+
+        public static ISet<string> SplitTags(string tags)
+        {
+            if (string.IsNullOrEmpty(tags))
+            {
+                return new HashSet<string>();
+            }
+
+            return tags.Split(',', StringSplitOptions.RemoveEmptyEntries).ToHashSet();
```

Comment:
```suggestion
            return tags.Split(',', StringSplitOptions.RemoveEmptyEntries).ToHashSet(StringComparer.Ordinal);
```

assuming casing should matter e.g. `ThisTag` != `thisTag`